auth/src/main/java/com/firebase/ui/auth/ui/idp/CredentialSignInHandler.java

```diff
@@ -94,12 +95,6 @@ public void onFailure(@NonNull Exception e) {
     }
 
     private class StartWelcomeBackFlow implements OnSuccessListener<ProviderQueryResult> {
```

Comment:
This class isn't static so I'm not sure why we passed in the email. Seems like extra boilerplate to me.